### ISSUE TITLE
Rule to indent multiline strings

### ISSUE THREAD

**Issue by hartbit** [nicklockwood/SwiftFormat#243]
It would be nice to have a rule to indent multiline strings from:

```swift
let sql = """
SELECT *
FROM authors
WHERE authors.name LIKE '%David%'
"""
```

into

```swift
let sql = """
    SELECT *
    FROM authors
    WHERE authors.name LIKE '%David%'
    """
```

It seems to be the style most often used in the community too.

**Comment by nicklockwood** [comment]
@hartbit sorry for not responding earlier. Seems like a reasonable feature - would you be interested in creating a PR?

**Comment by hartbit** [comment]
I have very little left for Open Source contributions :-( I'll have a look but I can't promise anything.

**Comment by neilt** [comment]
Also note that indentation inside the multiline string should be preserved.

For example:
```swift
let sql = """
    SELECT *
    FROM  authors,
          books
    WHERE authors.name LIKE '%David%'
         AND pubdate < $1
    """
```

**Comment by nicklockwood** [comment]
@neilt yes, that logic is already in place.

The SwiftFormat tokenizer actually bakes the relative indent into the string tokens themselves, so as long as the string lines are all given the same indent, the relative indent will be correctly preserved.

**Comment by nicklockwood** [comment]
@facumenzella my understanding of the feature request was that it should convert:

```swift
let sql = """
SELECT *
FROM  authors,
      books
WHERE authors.name LIKE '%David%'
     AND pubdate < $1
"""
```

to:

```swift
let sql = """
    SELECT *
    FROM  authors,
          books
    WHERE authors.name LIKE '%David%'
         AND pubdate < $1
    """
```

Which I think should not be a new rule, but a configuration option like `--indentstrings true` that is applied by the existing `indent` rule.

Unwrapping the `"""` onto previous line seems like a different requirement.

**Comment by nicklockwood** [comment]
@hartbit implemented in 0.49.0
